tests/src/build.rs

```diff
@@ -85,6 +85,11 @@ fn main() {
         .compile_protos(&[src.join("well_known_types.proto")], includes)
         .unwrap();
 
+    config
+        .unknown_fields_message("unknown_fields.MessageWithUnknownFields")
```

Comment:
This string should start with a dot, as that is the match notation used in other `Config` functions.